packages/@lwc/template-compiler/src/codegen/index.ts

```diff
@@ -568,7 +568,9 @@ function generateTemplateFunction(codeGen: CodeGen): t.FunctionDeclaration {
     return t.functionDeclaration(
         t.identifier(TEMPLATE_FUNCTION_NAME),
         args,
-        t.blockStatement(body)
+        t.blockStatement(body, {
+            trailingComments: [t.comment(`LWC compiler v${COMPILER_VERSION_NUMBER}`)],
+        })
```

Comment:
I tried `trailingComments`, `innerComments`, and `leadingComments` on both the block statement and the function declaration. This is the only one that worked.

I also tried adding some padding around the string (i.e. `/* LWC compiler v1 */`, not `/*LWC compiler v1*/`), but that didn't work either. I assume this is some limitation of `astring`. In any case, it's not a big deal.

Reply:
It's because `astring` is calling `reindent`: https://github.com/davidbonnet/astring/blob/9cb3964cfb508c1c25a5bfa0eff4fe14f25c7046/src/astring.js#L194